app/src/styles.sass

```diff
@@ -14,7 +14,8 @@ body
 .main-container
   max-width: $contmaxwidth
   margin: 6em auto 0
-  padding: 0 $medium-space
```

Comment:
This has nothing to do with the current ticket. This proposition could be discussed later. This is not a detail.

Reply:
PTAL
Because the information part is too long, so the main container could occupy the Navtittle part, I delete padding-bottom in the first time. 
This time I keep this, we can adjust it until the information part is finished.

Reply:
padding-bottom was 0. Deleting the line has no effect on padding-bottom. The line just add horizontal padding (left and right).

Reply:
Ah, okay, I understand. I think it's because the increase of horizontal padding shortens the width of the information section, so that the main-container becomes longer, causing the Navtittle to not be fixed when I scroll the page. 
So we can discuss the horizontal padding after the information part is finished.

Reply:
There is a bug which appears when Firefox simulates Samsung S9. Maybe that's what you're writing about. I don't know if it is an intentional bug. Anyway, don't care about it.

Reply:
OK. Do I need any further modification for this ticket or I can start to work on another one ?